doc/src/manual/noteworthy-differences.md

```diff
@@ -224,6 +229,24 @@ For users coming to Julia from R, these are some noteworthy differences:
     This means it will overflow, such that `2^64 == 0`. If you need larger values use another appropriate type,
     such as `Int128`, [`BigInt`](@ref) or a floating point type like `Float64`.
+  * The imaginary unit `sqrt(-1)` is represented in Julia as `im`, not `j` as in Python.
+  * In Julia, the exponentiation operator is `^`, not `**` as in Python.
+  * Julia uses `nothing` of type `Nothing` to represent a null value, whereas Python uses `None` of type `NoneType`.
+  * In Julia, the standard operators over a matrix type are matrix operations, whereas, in Python, the standard operators are element-wise operations. When both `A` and `B` are matrices, `A * B` in Julia performs matrix multiplication, not element-wise multiplication as in Python. `A * B` in Julia is equivalent with `A @ B` in Python, whereas `A * B` in Python is equivalent with `A .* B` in Julia.
+  * The adjoint operator `'` in Julia returns an adjoint of a vector (a lazy representation of row vector), whereas the transpose operator `.T` over a vector in Python returns the original vector (non-op).
+  * Functions in Python have one implementation (no polymorphism), whereas in Julia a function may contain multiple concrete implementations (called *Methods*), selected via multiple dispatch.
+  * There are no classes in Julia. Instead they are structures (mutable or immutable), containing data but no methods. 
+  * Calling a method of a class in Python (`a = MyClass(x), x.func(y)`) corresponds to a function call in Julia, e.g. `a = MyStruct(x), func(x::MyStruct, y)`. In general, multiple dispatch is more flexible and powerful than the Python class system.
+  * Python classes can inherit from one or more superclasses. Julia structures may have exactly one abstract supertype. 
+  * The Python code structure is defined by directories (Packages) and files (Modules), whereas the logical Julia program structure (Packages and Modules) is independent of the file strucutre (`include` for additional files).
+  * Conditional expression in Python `1 if x > 0 else -1` corresponds to the Ternary operator `x > 0 ? 1 : -1` in Julia.
+  * In Python, the `@` symbol refers to a decorator, whereas in Julia it refers to a macro.
+  * Exception handling in Julia is done using `try` - `catch` - `finally`, instead of `try` - `except` - `finally`. In contrast to Python, it is not recommended to use exception handling as part of the normal workflow in Julia due to performance reasons.
+  * In Julia loops are fast, there is no need to write "vectorized" code for performance reasons.
+  * Non-constant global variables impose a significant performance loss in Julia and should therefore be avoided (see [Performance Tips](@ref man-performance-tips)). For Python, global variables are not associated with a performance penalty.  
+  * In Python, the majority of values can be used in logical context. In Julia, you need explicit conversion to Bool.
```

Comment:
re https://github.com/JuliaLang/julia/pull/35268#discussion_r399700341

@waldyrious Were you suggesting something like this?

```suggestion
  * In Python, the majority of values can be used in logical contexts (e.g, `1 if "a" else 0` evaluates to `1`). In Julia, you need explicit conversion to `Bool` (e.g., `"a" ? 1 : 0` throws).
```

@lungben What do you think?

Reply:
Yes, this example is helpful here.

Reply:
I'm not sure relying on another language difference to illustrate this would be a good idea. How about simply mentioning that `0` and `None` are evaluated as logical `False` and that conversely, any/most things, such as a simple `"a"`, act like `True`? I can't think of a compact way to say that but thought I'd leave the suggestion.

Reply:
Maybe use a simple `if` statement for illustration?

> In Python, the majority of values can be used in logical contexts (e.g,  for `if "a":` the following block is executed, for `if "":` not). In Julia, you need explicit conversion to `Bool` (e.g., `if "a"` throws an exception).

Reply:
Something like that sounds good, yeah :)

Reply:
Sounds good to me, too.

Reply:
Thanks, is updated.